docs/content/direct/binding.md

```diff
@@ -1,5 +1,179 @@
+  usage of the word "resource". In much --- but not all --- of the
+  code, the word "resource" is roughly analogous to a "kind" of object
+  --- and specifically _not_ an individual object. In some of the code
+  and much of the documentation, the word "resource" refers to an
+  individual object. Some authors hold with the wider web world, in
+  which the word "resource" means anything with a URI/URL (Kubernetes
+  object kinds _and_ individuals both have URLs). In this position in
+  KubeStellar we take the first view: by "resource" here we mean the
+  Kubernetes concept that is roughly analogous to "kind" and the names
+  are lowercase and plural. Examples of resources are `namespaces` and
+  `deployments`. KubeStellar also allows this list to include `*`
+  meaning to match every resource; in this case the list should _only_
+  contain that value. The empty array --- which is the default value
+  --- is treated specially, it matches every resource.
+
+- `namespaces`: an array of strings containing names of
+  namespaces. Again, a value of `*` matches every namespace and should
+  be the only thing in the array if it is present. `*` also matches
+  cluster-scoped objects.  The empty array --- which is the default
+  value --- is treated specially, it matches every object.
+
+- `namespaceSelectors`: an array of label selectors that test the
+  labels of a `Namespace` object --- specifically, the `Namespace`
+  object of the namespace of the workload object being tested. For
+  example, when a `Deployment` object has namespace `foo`, this test
+  is applied to the `Namespace` object named `foo`. The `Namespace`
+  object --- and thus the workload object having that namespace ---
+  passes this test if the `Namespace`'s labels pass _any_ of the label
+  selectors in this array. Again, the empty list is the default value
+  and is given the exceptional meaning of matching everything.
+
+- `objectSelectors`: another array of label selectors, this one
+  applying to the labels of the workload object in question. Again,
+  the workload object passes this test if the object's labels match
+  _any_ of the label selectors. Again, the empty list is the default
+  value and is given the exceptional meaning of matching everything.
+
+- `objectNames`: an array of object names. A workload object passes
+  this test if its name is in this array, or this array contains `*`,
+  or this array is empty (which is the default value).
+
+A `DownsyncPolicyClause` also contains the following two fields to
+modulate the downsync.
+
+- `statusCollectors`: an array of references to `StatusCollector`
+  objects that specify how status from the WECs is to be combined and
+  reported in `CombinedStatus` objects in the WDS. See [Combining
+  returned status](combined-status.md) for more details.
+
+- `createOnly`: a boolean that, when true, suppresses on-going
+  maintenance of the object in the WEC. When this boolean is `true`,
```

Comment:
suppresses on-going  maintenance of the object in the WEC by KubeStellar   
(you can still maintain it through the WEC)

Reply:
Moot now.